presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
+* Add BigQuery Connector Plugin.
```

Comment:
As per - we should create a new section `BigQuery Connector Changes` and then the bullet there can be `* Introduced BigQuery Connector. See :doc:`/connector/bigquery` to get started`